### ISSUE TITLE
The uppercase VulnerabilityAssessmentName returned by the DatabaseVulnerabilityAssessments_CreateOrUpdate API.

### ISSUE THREAD

**Issue by sinbai** [Azure/azure-rest-api-specs#17192]
When creating a [DatabaseVulnerabilityAssessments](https://github.com/Azure/azure-rest-api-specs/blob/0fe46ea84880c3232c7a86db0c73f30975b11b0b/specification/sql/resource-manager/Microsoft.Sql/preview/2021-05-01-preview/DatabaseVulnerabilityAssessments.json#L83) resource. The [value](https://github.com/Azure/azure-rest-api-specs/blob/0fe46ea84880c3232c7a86db0c73f30975b11b0b/specification/sql/resource-manager/Microsoft.Sql/preview/2021-05-01-preview/DatabaseVulnerabilityAssessments.json#L47) of VulnerabilityAssessmentName is fixed as "default" , but the name in response body is "Default". details:

The response body is:
```
{
  "properties": {
    "storageContainerPath": "https://accsa.blob.core.windows.net/testsc/",
    "recurringScans": {
      "isEnabled": false,
      "emailSubscriptionAdmins": true,
      "emails": []
    }
  },
  "id": "/subscriptions/xxx/resourceGroups/testRG-mssql/providers/Microsoft.Sql/servers/testsqlserver/databases/testdb/vulnerabilityAssessments/Default",
  "name": "Default",
  "type": "Microsoft.Sql/servers/databases/vulnerabilityAssessments"
}
```

Actual: The "name" in response body is "Default". The VulnerabilityAssessmentName of "id" in response body is "Default".
Expected: All VulnerabilityAssessmentName in response body should be "default".

**Comment by MasterKolka** [comment]
@ruowan will this be fixed soon? It broke managing Azure SQL VAs via Terraform. Thanks.

**Comment by catt231** [comment]
agreed this is forcing replacements of resources and creating noise in my pipelines I don't need... would love to see it resolved soon please

**Comment by krohm** [comment]
Any progress on that ?
